library/src/camera_impl.cxx

```diff
@@ -214,6 +214,17 @@ camera& camera_impl::pitch(angle_deg_t angle)
   return *this;
 }
 
+//----------------------------------------------------------------------------
+camera& camera_impl::ortoview()
```

Comment:
Maybe a better name would be `toggleOrthographic()` or `togglePerspective()` to clearly understand calling it again revert to the original state.